src/ixdat/measurements.py

```diff
@@ -320,119 +717,85 @@ def __add__(self, other):
-
-    for s in series_list:
-        if not (s.name == name and s.unit == unit and s.__class__ == cls):
-            raise BuildError(f"can't append {series_list}")
-        data = np.append(data, s.data)
-    data = data[sort_indeces]
-
-    return cls(name=name, unit_name=unit.name, data=data, tseries=tseries)
-
-
-def append_tseries(series_list, tstamp=None, sorted=True, return_sort_indeces=False):
-    """Return new TSeries with the data appended relative to series_list[0].tstamp"""
-    name = series_list[0].name
-    cls = series_list[0].__class__
-    unit = series_list[0].unit
-    tstamp = tstamp or series_list[0].tstamp
-    data = np.array([])
-
-    for s in series_list:
-        if not (s.name == name and s.unit == unit and s.__class__ == cls):
-            raise BuildError(f"can't append {series_list}")
-        data = np.append(data, s.data + s.tstamp - tstamp)
-
-    if sorted:
-        sort_indices = np.argsort(data)
-        data = data[sort_indices]
-    else:
-        sort_indices = None
-
-    tseries = cls(name=name, unit_name=unit.name, data=data, tstamp=tstamp)
-    if return_sort_indeces:
-        return tseries, sort_indices
-    return tseries
-
-
-def fill_object_list(object_list, obj_ids, cls=None):
-    """Add PlaceHolderObjects to object_list for any unrepresented obj_ids.
-
-    Args:
-        object_list (list of objects or None): The objects already known,
-            in a list. This is the list to be appended to. If None, an empty
-            list will be appended to.
-        obj_ids (list of ints or None): The id's of objects to ensure are in
-            the list. Any id in obj_ids not already represented in object_list
-            is added to the list as a PlaceHolderObject
-        cls (Saveable class): the class remembered by any PlaceHolderObjects
-            added to the object_list, so that eventually the right object will
-            be loaded.
-    """
-    cls = cls or object_list[0].__class__
-    object_list = object_list or []
-    provided_series_ids = [s.id for s in object_list]
-    if not obj_ids:
-        return object_list
-    for i in obj_ids:
-        if i not in provided_series_ids:
-            object_list.append(PlaceHolderObject(i=i, cls=cls))
-    return object_list
-
-
-def time_shifted(series, tstamp=None):
-    """Return a series with the time shifted to be relative to tstamp"""
-    if not tstamp:
-        return series
-    cls = series.__class__
-    if isinstance(series, TimeSeries):
-        return cls(
-            name=series.name,
-            unit_name=series.unit.name,
-            data=series.data + series.tstamp - tstamp,
-            tstamp=tstamp,
-        )
-    elif isinstance(series, ValueSeries):
-        series = cls(
-            name=series.name,
-            unit_name=series.unit.name,
-            data=series.data,
-            tseries=time_shifted(series.tseries, tstamp=tstamp),
-        )
-    return series
+class Calibration(Saveable):
```

Comment:
I've come across some extra information on this that may inform how we handle this, aux tables vs. flexible columns. What I've learned in the meantime is that I don't think it is as easy to change database layout as I think your design for this lean up against. It may be ok simple to add a whole table, but you will need a DB admin. In any case, let's save that for a discussion when we reach DB.